app/views/layouts/nav/_navbar.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar">
+  <div class="navbar-item">
+    <div class="home">
+      <%= link_to "Private Events", root_path %>
+    </div>
+  </div>
+  <% if user_logged_in? %>
```

Comment:
- [ ] You have a lot of logic in your views, the conditionals, make sure to refactor that to helpers, or partials because having ruby blocks on the views slows down the application.